region/brinstar/blue.json

```diff
@@ -30,7 +30,25 @@
           },
           "locks": [
             {
-              "unlock": [ "Kill All Enemies" ]
+              "lock": [ "ZebesAwake" ],
+              "unlock": [ 
+                {"obstacle":{
+                  "id": "C",
+                  "requires": [ 
```

Comment:
It's one of those things where we'll have to  sit down and really think about whats needed. There's a lot of ways this can be doable, including tanking some damage then breaking obstacle A to shoot them from behind it.
But that can be looked at later.